ringtones.md

```diff
@@ -7,7 +7,44 @@ body_id:    ringtones
 
 Better than the beep-boop sound your phone normally makes.
 
-- [IRL Ringtone (Android)](http://assets.mozilla.net/irlringtones/irl-ringtone-android.mp3)
-- [IRL Ringtone (Apple)](http://assets.mozilla.net/irlringtones/irl-ringtone-apple.m4r)
+- [IRL Ringtone (Android)](https://assets.mozilla.net/irlringtones/irl-ringtone-android.mp3)
+- [IRL Ringtone (Apple)](https://assets.mozilla.net/irlringtones/irl-ringtone-apple.m4r)
 
-Not quite sure how to set a custom ringtone? We've put together [some instructions](http://assets.mozilla.net/irlringtones/setting-a-custom-irl-ringtone.pdf) to help you out. (But you might need to search out additional instructions depending on your device.)
+Not quite sure how to set a custom ringtone? We've put together some instructions to help you out. (But you might need to search out additional instructions depending on your device.)
+
+##  Setting a custom IRL ringtone on your smartphone
+<p></p>
+### Apple iPhone
+
+**First, start on your computer**
+
+1. Download the file [irl-ringtone-apple.m4r](https://assets.mozilla.net/irlringtones/irl-ringtone-apple.m4r) from our site and save it somewhere that will be easy for you to drag and drop from
+2. Connect your iPhone to your computer with the lightning cable
+3. In iTunes, select the iPhone from the menu bar (top left)
+
+*If using iTunes 12:*
```

Comment:
Should these be `h4`s? Is there even a way to do `h4` in markdown? The single asterisk renders as `em` and that feels wrong, so maybe just lose that emphasis and go with bare paragraphs if h4 isn't an option.